.eslintignore

```diff
@@ -0,0 +1,34 @@
+# Ignore dist files
+client/dist/
+
+# Ignore legacy files
```

Comment:
Is this supposed to be here?

Reply:
Yes ... we don't want to lint the distribution files.

Reply:
I'm talking about the legacy files. They're referencing admin?

Reply:
Sorry. You're right, I copied this file from admin and there's no need for any legacy file exclusion, because they are no legacy files.

Reply:
I've removed all the extra exclusion rules and just kept the generic ones.